parameter_files/archive/apiX_pr1262.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
+<all>
+  <!-- This parameter patch file updates the parameter file with the necessary changes -->
+  <!-- per PR 995.  This makes the following changes: -->
+  <!-- - updates the meta data for the initial seed density -->
+
+  <!-- To use this patch file run the following command: -->
+
+  <!--   Run `python ../tools/UpdateParamAPI.py -\-f archive/api27.0.0_080923_luh2.xml` -->
+
+  <!-- To update a different CDL parameter file, change the file listed the `base_file`. -->
+  <!-- This requires that said file is up to date with API 25.5. -->
+
+  <!-- General xml file usage notes: -->
+  <!-- Special mod types "variable_add": creates a new variable
+		 "dimension_add": creates a new dimension
+		 "variable_del": removes a variable, must include attribute "name" 
+		 "variable_change": change values in the parameter, possibly including the name, and
+                          this must also come with a name attribute as well -->
+  <!-- To change a dimension. The easiest way to script this, is to delete the variables
+		 that use it, delete the dimension. Re-add it with the new size, and then re-add
+		 the variables -->
+  <!-- -->
+  
+  <base_file>archive/base.cdl</base_file>
+  <new_file>fates_params_default.cdl</new_file>
```

Comment:
I'm assuming this will be updated closer to integration when we'll have a better idea what the default parameter file looks like at that time.

Reply:
I haven't introduced these elsewhere yet.  The variable add could come in early, but the variable deletions needs to happen with this PR or later, correct?

Reply:
New variables added via ed07677b15ff46badb8218a5691a0396694cf40d.  We'll catch up the deletions later.